### ISSUE TITLE
Piquage pro3 plus

### ISSUE THREAD

**Issue by coutud** [jayofelony/pwnagotchi#169]
Do you know how I can use this battery and the level indicator working
Thanks

**Comment by jayofelony** [comment]
Yes it does, search for pisugar3 pwnagotchi plugin on google. And manually install the following package;

`sudo apt-get install python3-smbus`

**Comment by ChanceVegas** [comment]
> Yes it does, search for pisugar3 pwnagotchi plugin on google. And manually install the following package;
> 
> `sudo apt-get install python3-smbus`

Hello, does the latest release also support the gpio buttons for the pisugar3 plus?

**Comment by rai68** [comment]
Closing due to inactivity, please open a new one if your issue persists.
